recipes/cpputest/all/conanfile.py

```diff
@@ -0,0 +1,100 @@
+    name = "cpputest"
+    description = \
+"CppUTest is a C /C++ based unit xUnit test framework for unit testing and for test-driving your code." \
+"It is written in C++ but is used in C and C++ projects and frequently" \
+"used in embedded systems but it works for any C/C++ project."
+    license = "BSD-3-Clause License"
+    topics = ("conan", "testing", "unit-testing")
+    homepage = "http://cpputest.github.io"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "verbose": [True, False],
+        "use_std_c_lib": ["ON", "OFF"],
+        "use_std_cpp_lib": ["ON", "OFF"],
+        "use_cpp11": ["ON", "OFF"],
+        "detect_mem_leaks": ["ON", "OFF"],
+        "extensions": ["ON", "OFF"],
+        "longlong": ["ON", "OFF"],
+        "coverage": ["ON", "OFF"],
+        "tests": ["ON", "OFF"],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": False,
+        "verbose": False,
+        "use_std_c_lib": "ON",
+        "use_std_cpp_lib": "ON",
+        "use_cpp11": "ON",
+        "detect_mem_leaks": "ON",
+        "extensions": "ON",
+        "longlong": "ON",
+        "coverage": "OFF",
+        "tests": "OFF",
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename(self.name + "-" + self.version, self._source_subfolder)
+
+    def build(self):
+        # tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+        #                       "set (CMAKE_POSITION_INDEPENDENT_CODE ON)", "")
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.verbose = self.options.verbose
+        self._cmake.definitions["STD_C"] = self.options.use_std_c_lib
+        self._cmake.definitions["STD_CPP"] = self.options.use_std_cpp_lib
+        self._cmake.definitions["C++11"] = self.options.use_cpp11
+        self._cmake.definitions["MEMORY_LEAK_DETECTION"] = self.options.detect_mem_leaks
+        self._cmake.definitions["EXTENSIONS"] = self.options.extensions
+        self._cmake.definitions["LONGLONG"] = self.options.longlong
+        self._cmake.definitions["COVERAGE"] = self.options.coverage
+        self._cmake.definitions["TESTS"] = self.options.tests
```

Comment:
I do not think consumer from CCI need this to be tested or produce coverage results...

https://github.com/cpputest/cpputest/blob/d14505cc9191fcf17ccbd92af1c3409eb3969890/CMakeLists.txt#L73

Reply:
Thanks. I think I've addressed this in a43fa98c59c858b3a66c1aec97037208ba0c7b3e - is that what you meant?